app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/triage/CommonTriageQuestionsController.scala

```diff
@@ -683,7 +648,7 @@ object CommonTriageQuestionsController {
       mapping(
         "individualUserType" -> of(
           FormUtils.radioFormFormatter(
-            if (periodOfAdminEnabled) options :+ PersonalRepresentativeInPeriodOfAdmin else options
+            options :+ PersonalRepresentativeInPeriodOfAdmin
```

Comment:
you could just stick `PersonalRepresentativeInPeriodOfAdmin` in the `Lists` in `options` rather than appending here. There's no reason to append here separately now